src/main/java/mate/academy/dao/jdbc/ManufacturerDaoJdbc.java

```diff
@@ -0,0 +1,117 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.ManufacturerDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Manufacturer;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoJdbc implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturers (manufacturer_name, manufacturer_country)"
+                + " VALUES (?,?);";
+        String querySelectAll = "SELECT * FROM manufacturers;";
+
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
```

Comment:
```suggestion
                PreparedStatement statement = connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
```
So you will be able to use: 
statement.executeUpdate();
            ResultSet resultSet = statement.getGeneratedKeys();